src/pages/enterprise/auth-connect.md

```diff
@@ -5,7 +5,7 @@ version: 2.2.0
 minor: 2.2.X
 ---
 
-Ionic Auth Connect handles logging in and/or registering a user with an authentication provider (such as Auth0, Azure AD, or AWS Cognito) using industry standard OAuth/OpenId Connect on iOS, Android, or on the web.
+Ionic Auth Connect handles logging in and/or registering a user with an authentication provider (such as Auth0, Azure AD, AWS Cognito, or Okta) using industry standard OAuth/OpenId Connect on iOS, Android, or on the web.
```

Comment:
Unfortunately the process to update this readme is a bit wonky. It lives in the AC repo (https://github.com/ionic-team/enterprise-auth-connect) and needs to be updated there first, otherwise these changes could get overwritten down the line. the native team uses that as the source of truth.

Reply:
that shouldn't hold us back from shipping this now, though :)